lsp-dart.el

```diff
@@ -65,6 +88,27 @@ imported into the current file. Defaults to true"
      ,(expand-file-name (f-join lsp-dart-sdk-dir "bin/snapshots/analysis_server.dart.snapshot"))
      "--lsp")))
 
+(defun lsp-dart--handle-closing-labels (_workspace params)
+  "Closing labels notification handling.
+PARAMS closing labels notification data sent from WORKSPACE."
+  (-let* (((&hash "uri" "labels") params)
+          (buffer (get-file-buffer (string-remove-prefix "file://" uri))))
```

Comment:
use `lsp--uri-to-path`. Also, I suggest checking if the buffer is still open to avoid reopening the buffer if it has been closed.

Reply:
Do you mean checking if the buffer still exists?

Reply:
I found `lsp--buffer-for-file`, I think this is the right t apply here, right?

Reply:
> I found `lsp--buffer-for-file`, I think this is the right t apply here, right?

Yes. You should check if the buffer is nil before using it.

Reply:
Done!